wpeverest/user-registration
# UR-346 Enhance - Parse smart tag values in appropriate format to display

### All Submissions:






### Changes proposed in this Pull Request:
Previously when smart tags for individual fields for eg. {{file_12345}}, {{profile_pic_url}}, etc were used then it would parse their value from the database and directly send or show to the user which was not in the appropriate format as their attachment_ids were shown instead of URLs. This PR fixes such issues relating to all fields not only shown in the example.

### How to test the changes in this Pull Request:

1. Create a form using all fields.
2. Use all the fields' smart tags to show in email or in customize my account addon endpoint contents ( [Smart Tags Support PR](https://github.com/wpeverest/user-registration-customize-my-account/pull/19) )
3. Register a user and see if appropriate values are displayed in appropriate format in email or endpoint contents.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Parse smart tag values in appropriate format to display.
